File: C/PRAK204-2310817310004-MSAMILRENDYNORSALEH.c
#include <stdio.h>

int main() {
    float j, t;
    float pi = 3.142857;
    printf("");
    scanf("%f", &j);
    printf("");
    scanf("%f", &t);
    float volume = pi * j * j * t;
    float luas = 2 * pi * j * (j + t);
    float keliling = 2 * pi * j;
    printf("Volume = %.2f\n", volume);
    printf("Luas = %.2f\n", luas);
    printf("Keliling = %.2f\n", keliling);

    return 0;
}

File: Python/PRAK204-2310817310004-MSAMILRENDYNORSALEH.py
j, t = map(int, input("").split())
pi = 22/7
volume = pi * j**2 * t 
luas = 2 * pi * j * (j + t)
keliling = 2 * pi * j
print("Volume = {:.2f}".format(volume)) 
print("Luas = {:.2f}".format(luas))
print("Keliling = {:.2f}".format(keliling))
